src/main/java/com/flightstats/hub/app/ShutdownManager.java

```diff
@@ -51,7 +56,9 @@ protected void shutDown() throws Exception {
 
     public boolean shutdown(boolean useLock) throws Exception {
         logger.warn("shutting down!");
-        getMetricsService().mute();
+        String[] tags = { "restart", "shutdown" };
+        statsdReporter.event("Hub Restart Shutdown", "shutting down", tags);
```

Comment:
Wait, it used to mute metrics and then try to report that it was shutting down on that service?

Reply:
Yep, that's exactly what it seems was happening.
The mute method hits the api endpoint at datadog to sleep monitoring on the host for a few minutes during restarts
We don't specify a start time in out api request so it would do the default and immediately mute, making the later call to event useless :)

https://docs.datadoghq.com/api/?lang=python#schedule-monitor-downtime

Reply:
Aha. Thanks for fixing it. :)